homeassistant/components/iqvia/sensor.py

```diff
@@ -106,8 +106,8 @@ class ForecastSensor(IQVIAEntity):
         if not self._iqvia.data:
             return
 
-        data = self._iqvia.data[self._type].get('Location')
-        if not data:
+        data = self._iqvia.data[self._type]['Location']
```

Comment:
@bachya I'm seeing an error related to this change.  Reverting it fixes the sensors for me.


```
2019-05-16 22:33:03 ERROR (MainThread) [homeassistant.components.sensor] iqvia: Error on device update!
Traceback (most recent call last):
  File "/usr/src/app/homeassistant/helpers/entity_platform.py", line 261, in _async_add_entity
    await entity.async_device_update(warning=False)
  File "/usr/src/app/homeassistant/helpers/entity.py", line 375, in async_device_update
    await self.async_update()
  File "/config/custom_components/iqvia/sensor.py", line 109, in async_update
    data = self._iqvia.data[self._type]['Location']
KeyError: 'Location'
```

Reply:
Ugh, thanks @stbenjam. I'll address.

Reply:
Thanks so much! Otherwise looks like it's all working for me, nice job on this component

Reply:
@stbenjam Could you test for me? https://github.com/home-assistant/home-assistant/pull/23931